src/reducers/getBookmarkReducer.js

```diff
@@ -0,0 +1,43 @@
+import { getBookmarkType } from '../actions/getBookmark';
+
+const initialState = {
+  loading: false,
+  articles: [],
+  page: {
+    first: 1,
+    current: 1,
+    last: 1,
+    currentCount: 0,
+    totalCount: 0,
+    description: ''
+  },
+  message: '',
+};
+
+/**
+ * reset password action creator
```

Comment:
**Nitpick**

- This isn't an action creator is it?
- I believe this has nothing to do with reset password

Reply:
@devdbrandy resolved